.github/workflows/check-spelling.yml

```
@@ -0,0 +1,44 @@
+name: Check Spelling
+
+on:
+  pull_request:
+  push:
+    branches:
+      - dev
+  workflow_call:
+    inputs:
+      dictionary_source_prefixes:
+        description: JSON map of prefixes for dictionary urls, "cspell" is necessary
```

Same for this and the other parameter in the action.

```suggestion
        description: JSON map of prefixes for dictionary URLs, "cspell" is necessary.
```